### ISSUE TITLE
Can't update lottie to version 3

### ISSUE THREAD

**Issue by ShafiqSadat** [Ezaldeen99/material_dialogs#33]
``` 
Because material_dialogs 1.1.4 depends on lottie ^2.1.0 and no versions of material_dialogs match >1.1.4 <2.0.0, material_dialogs ^1.1.4 requires lottie ^2.1.0.
So, because sarafi depends on both lottie ^3.1.0 and material_dialogs ^1.1.4, version solving failed.


You can try the following suggestion to make the pubspec resolve:
* Consider downgrading your constraint on lottie: flutter pub add lottie:^2.7.0

**Comment by Ezaldeen99** [comment]
please resolve the conflict in your PR

**Comment by Ezaldeen99** [comment]
closed as fixed in #35
